.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 **/*.rs.bk
+.idea
```

Comment:
I've been avoiding adding editor-specific files to .gitignore. [Consider setting up a global .gitignore file.](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
I check in non-volatile .idea files in a lot of my other projects so a global would be problematic. But I can manually remove it from PRs when I open them.